### PR TITLE
CVSL-1357 Updating packages

### PR DESCRIPTION
**Pull request by andrewrlee**
Will require `npm install npm@10`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -40,8 +40,7 @@ jobs:
       - checkout
       - run:
           name: Update npm
-          # temporary hard code version until we move to v10
-          command: 'sudo npm install -g npm@9.8.1'
+          command: 'sudo npm install -g npm@10'
```
Comment (andrewrlee) [inline .circleci/config.yml:43]: npm 10 is properly released now

File: server/listeners/eventHandlers/domainEvents/index.ts [hunk 2]
```diff
@@ -32,6 +32,9 @@ export default function buildEventHandler({ licenceService, prisonerService }: S
             offenderDetailsChangedEventHandler.handle(eventMessage).catch(error => logger.error(error))
           }
           break
+        default: {
+          // silently ignore
+        }
```
Comment (andrewrlee) [inline server/listeners/eventHandlers/domainEvents/index.ts:37]: one of the eslint rules is to warn if no default case in switch statements - this is generally a good thing. 

I've left the existing behaviour (ignore) for now but at least its explicit.

**Review by justin-gardiner — APPROVED** [review]
lgtm

**Review by justin-gardiner — APPROVED** [review]
lgtm

**Pull request merged**